### DISCUSSION TITLE
Revenue script extensions gives 404

### DISCUSSION THREAD

**Discussion by pasqualevitiello** [plausible/analytics#3314] [Self-Hosted Support]
I am trying to track revenues on a fresh new install. When I add the revenue extension to the script I get a 404:

https://plausible.website.com/js/script.revenue.js -> 404

All the other scripts work, ie:

- https://plausible.website.com/js/script.js
- https://plausible.website.com/js/script.exclusions.js
- https://plausible.website.com/js/script.exclusions.hash.js
- etc

Anything I've been missing?

**Comment by ruslandoga** [comment]
👋 @pasqualevitiello 

AFAIK revenue metrics haven't been released in v2.0.0.
